<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battleWidget</class>
 <widget class="QWidget" name="battleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="heroLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>hero</string>
   </property>
  </widget>
  <widget class="QLabel" name="monsterLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>monster</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="heroHpBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="value">
    <number>90</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="monsterHpBar">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="value">
    <number>48</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QLabel" name="logLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正舒体</family>
     <pointsize>16</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>史莱姆使用了粘液攻击！！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="baseAttackButton">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>515</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>普通攻击</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skill3Button">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>515</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PASS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skill2Button">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>515</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>次级治疗术</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skill1Button">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>515</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>炎爆术</string>
   </property>
  </widget>
  <widget class="QPushButton" name="itemButton">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>485</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>道具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fleedButton">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>565</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>逃跑</string>
   </property>
  </widget>
  <widget class="QLabel" name="vsLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Broadway</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>V S</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>显示结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="demonstrationLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Imprint MT Shadow</family>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示伤害/治疗数字</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
